app/Platform/Actions/BuildHubBreadcrumbsAction.php

```diff
@@ -447,6 +448,20 @@ private function findSameTypeParent(GameSet $gameSet, string $type, array $visit
             if ($sameEventParent) {
                 return $sameEventParent;
             }
+
+            // As a fallback, look for ANY parent hub that is an Events hub.
+            // This handles cases where naming is inconsistent (eg: "[RA Roulette 2022 - X]" with parent "[Events - RA Roulette (RAWR)]").
+            $anyEventParent = $gameSet->parents()
+                ->select('game_sets.*')
+                ->where('game_sets.type', GameSetType::Hub)
+                ->where('game_sets.title', 'like', '[Events - %')
+                ->whereNotIn('game_sets.id', $visited)
+                ->whereNull('game_sets.deleted_at')
+                ->first();
```

Comment:
Should the events themselves have a similar breadcrumb?
http://localhost:64000/event/80-ra-roulette-2022